src/index.html

```diff
@@ -11,7 +11,174 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application" 
+      method="post"
+    >
+      <fieldset>
+        <legend>Personal information</legend>
+        <div class="form-field">
+          <label for="surname">Surname:</label>
+          <input 
+          type="text"
+          id="surname"
+          name="surname"
+          autocomplete="off"
+          >
```

Comment:
Not fixed 
<img width="1015" alt="image" src="https://user-images.githubusercontent.com/99403198/214264992-3f7d6c42-c64d-4c34-83cc-f4120fe8f366.png">
```suggestion
          <input 
            type="text"
            id="surname"
            name="surname"
            autocomplete="off"
          >
```